databases/db.js

```diff
@@ -91,5 +105,5 @@ const random = sequelize.random();
 const { Op } = Sequelize;
 
 module.exports ={
-    User, Sticker, Event, Promotion, ad, game, team, random, Op, inventory
+    User, Sticker, Event, Promotion, ad, game, team, random, Op, inventory, money
```

Comment:
Mayusculas para los modelos pls
```suggestion
    User, Sticker, Event, Promotion, ad, game, team, random, Op, inventory, Money
```